globals.go

```diff
@@ -291,6 +291,9 @@ const (
 	// Valid types of performing git clone
 	CloneMethodLegacySSH = "legacy-ssh"
 	CloneMethodOAuth     = "oauth"
+
+	// PublicRepoName is the repo name for which a project ref is considered public.
+	PublicRepoName = "wiredtiger"
```

Comment:
I know it's a one-off exception, but it is a little hackish to hard-code the one special repo. Is the plan to keep this until the rest of time, or is there a plan to get them to use auth?

Reply:
Rather than this, I'd rather we just have wiredtiger keep Private set to true. We aren't sure yet I think what the longterm plan is for wiredtiger or if other projects will end up having an argument for this treatment.

Reply:
Sounds fair enough, I suppose I got too excited at the concept of deleting a bunch of code centered around `private`. Reverted it back to having this flag, all instances of this `PublicRepoName` are gone. This change will now come with a proposed prod DB update needing LGTM in the ops channel to set all projects to private except the 5-6 wiredtiger projects.

Reply:
Before making the DB update we can also probably follow up with Alex on the ticket to see if he can confirm if this should be considered for all wiredtiger projects.